package.json

```diff
@@ -65,7 +63,8 @@
     "tsup": "7.2.0",
     "turbo": "1.12.0",
     "typescript": "^5.2.2",
-    "yorkie": "^2.0.0"
+    "yorkie": "^2.0.0",
+    "tiny-invariant":"^1.3.1"
```

Comment:
Why do we need tiny-invariant here?

Reply:
ah, in that case we should add it to the example's `package.json`. Otherwise it will be missing if you scaffold a project based on an example.